src/_components/AuthenticateComponent.jsx

```diff
@@ -1,14 +1,23 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 import { Route, Redirect } from 'react-router-dom';
 
-const AuthenticateComponent = ({ component: Component, isAuthenticated, ...options }) => {
-  return (
-    <Route {...options} render={props => (
-        isAuthenticated
-            ? <Component {...props} />
-            : <Redirect to={{ pathname: '/', state: { from: props.location } }} />
-    )} />
-  )
+const AuthenticateComponent = ({ component: Component, isAuthenticated, ...options }) => (
+  <Route
+    {...options}
+    render={props => (
+      isAuthenticated ?
```

Comment:
Please restructure the ternary to:
```
isAthenticated
  ? <Component {...props} />
  : <Redirect to={{ pathname: '/', state: { from: props.location } }} />
```
You can see the better example in the [airbnb nested ternaries](https://github.com/airbnb/javascript#comparison--nested-ternaries)

Reply:
Thanks @sachungo 
Yes. Those are the same addressed in this PR. We had one file having several hound errors that is why they seem to be more.